src/Apps/Search/SearchApp.tsx

```diff
@@ -26,11 +26,21 @@ export interface Props extends RouterState {
 const TotalResults: React.SFC<{ count: number; term: string }> = ({
   count,
   term,
-}) => (
-  <Serif size="5">
-    {count.toLocaleString()} Result{count > 1 ? "s" : ""} for "{term}"
-  </Serif>
-)
+}) => {
+  const formatResults = () =>
+    count.toLocaleString() +
+    " Result" +
+    (count > 1 ? "s" : "") +
+    " for " +
+    `"${term}"`
```

Comment:
Would a template string be cleaner here?

Reply:
Yup, much cleaner! fixed.